src/templates/add.svg

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="iso-8859-1"?>
```

Comment:
we are not allowed to push to the master not optimized svgs. You can use very simple package `svgo` :)